File: Jr/2021/BoilingWater.py
# Owen Bélanger-Love

# Get the temperature
t= int(input())

# Calculate the pressure
p = 5 * t - 400

# Print the temperature
print(p)

# Check if the pressure is below sea level
if p > 100:
    print(-1)

# Check if the pressure is at sea level
elif p == 100:
    print(0)

# The pressure is above sea level
else:
    print(1)

File: Jr/2021/BoilingWater.java
// Owen Bélanger-Love

// Import Scanner class
import java.util.Scanner;

public class BoilingWater {
    public static void main(String[] args) {

        // Create input Scanner
        Scanner input = new Scanner(System.in);

        // Declare variables
        int t, p;
    
        // Get the boiling temperature of water
        t = input.nextInt();
    
        // Caluclate the pressure
        p = 5 * t - 400;
    
        // Print the pressure
        System.out.println(p);
    
        // Check if the pressure is below sea level
        if (p > 100) {
            System.out.println(-1);
        }
    
        // Check if the pressure is at sea level
        else if (p == 100) {
            System.out.println(0);
        }
    
        // The pressure is above sea level
        else{
            System.out.println(1);
        }

        // Close input Scanner
        input.close();
    }
}
